src/Atmos/Model/turbulence.jl

```diff
@@ -131,3 +131,49 @@ end
 function scaled_momentum_flux_tensor(m::SmagorinskyLilly, ρν, S)
   (-2*ρν) * S
 end
+
+"""
+  Vreman{DT} <: TurbulenceClosure
+  
+  §1.3.2 in CLIMA documentation 
+Filter width Δ is the local grid resolution calculated from the mesh metric tensor. A Smagorinsky coefficient
+is specified and used to compute the equivalent Vreman coefficient. 
+
+1) ν_e = √(Bᵦ/(αᵢⱼαᵢⱼ)) where αᵢⱼ = ∂uᵢ∂uⱼ with uᵢ the resolved scale velocity component.
+2) βij = Δ²αₘᵢαₘⱼ
```

Comment:
Characters are not rendering here..

Reply:
Noted, thanks! 
- [ ] Switch to standard characters